packages/@ama-sdk/core/src/plugins/abort/abort.fetch.ts

```diff
@@ -0,0 +1,54 @@
+import type { FetchCall, FetchPlugin, FetchPluginContext, RequestOptions } from '../core';
+
+interface AbortCallbackParameters {
+  /** URL targeted */
+  url: string;
+
+  /** Fetch call options */
+  options: RequestInit | RequestOptions;
+
+  // TODO Now supported for all the modern browsers - should become mandatory in @ama-sdk/core@10.0
```

Comment:
we are in Otter v10 already :)

Reply:
then we forgot to apply this breaking change, will be for v11 then